static/resources_list.txt

```diff
@@ -53,3 +35,7 @@ templates/url_of_search_results_css
 opensearchdescription.xml
 ft_opensearchdescription.xml
 catalog_v2_searchdescription.xml
+skin/css/autoComplete.01.css
+skin/css/autoComplete.02.css
+skin/css/autoComplete.css
```

Comment:
These two  (`autoComplete.css` and `autoComplete.02.css`) are not used. Why not keeping only `autoComplete.01.css` but renaming it to `autoComplete.css`?

Reply:
Yes that's right, were bothering me since I took over this PR. Done, thanks.